docs/new-architecture-library-intro.md

```diff
@@ -48,11 +50,29 @@ export interface Spec extends TurboModule {
 export default (TurboModuleRegistry.get<Spec>('<MODULE_NAME>'): ?Spec);
 ```
 
+##### Turbo Module: TypeScript Example
+
+```ts
+import type { TurboModule } from 'react-native/Libraries/TurboModule/RCTExport';
+import * as TurboModuleRegistry from 'react-native/Libraries/TurboModule/TurboModuleRegistry';
```

Comment:
My TypeScript knowledge is very limited at the moment, but shouldn't we be able to import the types without the full path? Something like:

```suggestion
import type { TurboModule } from 'react-native';
import * as TurboModuleRegistry from 'react-native';
```

Reply:
+1

Reply:
Good point! Looks like both of these are exported from the main d.ts file.